package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "data-structures-and-algorithms",
+  "version": "1.0.0",
+  "description": "# Reverse an Array <!-- Short summary or background information --> reverse an array provided by the user",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
You can to get of the test script that you have in here and replace it with:
`    "test": "jest --coverage --verbose",
    "watch": "jest --watchAll" `